doc/when-things-go-wrong.rst

```diff
@@ -142,13 +142,16 @@ or just to learn how the runtime works.
 
 Normally the debug progress messages go to standard output.
 If the ``--windowed`` option is used when bundling a Windows app,
-they are displayed as MessageBoxes.
+they are sent to any attached debugger. If you don't have or are not using a
+debugger the DebugView_ the free (beer) tool can be used to display such
+messages. It must be run before the application being bundled.
```

Comment:
This is not obvious to me. I assume "It has be be started before the running the bundled application."

Reply:
Ok.